examples/lighting-app/efr32/src/AppTask.cpp

```diff
@@ -137,9 +137,10 @@ int AppTask::Init()
     uint32_t setupPinCode;
     std::string QRCode;
 
-    if (GetQRCode(setupPinCode, QRCode, chip::RendezvousInformationFlags::kBLE) == CHIP_NO_ERROR)
+    ConfigurationMgr().LogDeviceConfig();
```

Comment:
Should we log config only when the display is enabled? I would probably move it above.